src/pages/workspace/expensifyCard/WorkspaceEditCardLimitTypePage.tsx

```diff
@@ -38,20 +41,42 @@ function WorkspaceEditCardLimitTypePage({route}: WorkspaceEditCardLimitTypePageP
     const defaultLimitType = areApprovalsConfigured ? CONST.EXPENSIFY_CARD.LIMIT_TYPES.SMART : CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY;
     const initialLimitType = card?.nameValuePairs?.limitType ?? defaultLimitType;
     const promptTranslationKey =
-        initialLimitType === CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY
-            ? 'workspace.expensifyCard.changeCardMonthlyLimitTypeWarning'
-            : 'workspace.expensifyCard.changeCardSmartLimitTypeWarning';
+        initialLimitType === CONST.EXPENSIFY_CARD.LIMIT_TYPES.MONTHLY || initialLimitType === CONST.EXPENSIFY_CARD.LIMIT_TYPES.FIXED
+            ? 'workspace.expensifyCard.changeCardSmartLimitTypeWarning'
+            : 'workspace.expensifyCard.changeCardMonthlyLimitTypeWarning';
 
     const [typeSelected, setTypeSelected] = useState(initialLimitType);
     const [isConfirmModalVisible, setIsConfirmModalVisible] = useState(false);
 
+    const fetchCardLimitTypeData = useCallback(() => {
+        Policy.openPolicyEditCardLimitTypePage(policyID, Number(cardID));
+    }, [policyID, cardID]);
+
+    useFocusEffect(fetchCardLimitTypeData);
+
     const updateCardLimitType = () => {
         // TODO: add API call when it's supported https://github.com/Expensify/Expensify/issues/407833
     };
 
```

Comment:
This can be removed right?
```suggestion
```

Reply:
But we don't have an update yet, or why do we need to remove it?
This PR was to add an API call to get data and to correctly show modals/options, but updateCardLimitType is not implemented yet

Reply:
Ah yeah I mixed things up, thanks!